### PR TITLE
refactor: remove duplicated `that` word

### PR DESCRIPTION
**Pull request by alexprut**
Self explanatory.

### CODE REVIEW

File: content/concepts/consensus-network/amendments/known-amendments.ja.md [hunk 1]
```diff
@@ -450,7 +450,7 @@ XRP Ledgerの分散型取引所において、オファーの掛け合わせの
 | 1F4AFA8FA1BC8827AD4C0F682C03A8B671DCDF6B5C4DE36D44243A684103EF88 | 開発中 |
 
 Allows validators to include a new optional field in their validations to attest to the hash of
-the latest ledger that that validator considers to be fully validated. The consensus process can use this information to increase the robustness of consensus.
+the latest ledger that validator considers to be fully validated. The consensus process can use this information to increase the robustness of consensus.
```
Comment (mDuo13) [inline content/concepts/consensus-network/amendments/known-amendments.ja.md]:
```suggestion
the latest ledger that the validator considers to be fully validated. The consensus process can use this information to increase the robustness of consensus.
```

File: content/concepts/consensus-network/amendments/known-amendments.md [hunk 2]
```diff
@@ -479,7 +479,7 @@ This is a previous version of the [Flow](#flow) amendment. It was [rejected due
 | 1F4AFA8FA1BC8827AD4C0F682C03A8B671DCDF6B5C4DE36D44243A684103EF88 | In Development |
 
 Allows validators to include a new optional field in their validations to attest to the hash of
-the latest ledger that that validator considers to be fully validated. The consensus process can use this information to increase the robustness of consensus.
+the latest ledger that validator considers to be fully validated. The consensus process can use this information to increase the robustness of consensus.
```
Comment (mDuo13) [inline content/concepts/consensus-network/amendments/known-amendments.md]:
```suggestion
the latest ledger that the validator considers to be fully validated. The consensus process can use this information to increase the robustness of consensus.
```

**Review by mDuo13 — CHANGES_REQUESTED** [review]
This is not actually a typo, but is just an [awkward sentence in English](https://english.stackexchange.com/a/3426).

I've suggested a change that I think makes it read a little more naturally while maintaining the intended meaning.

**Comment by alexprut** [comment]
@mDuo13 thanks for the feedback, I've applied the suggested changes.

**Comment by alexprut** [comment]
@mDuo13 please let me know if you wanna proceed with this change, otherwise feel free to close this pr.

**Pull request merged**